packages/react-ui/src/components/HatchifyEverything/HatchifyEverything.tsx

```diff
@@ -27,6 +27,26 @@ export interface HatchifyEverythingProps<
 
 function HatchifyEverything<
   const TSchemas extends Record<string, PartialSchema>,
+>({ finalSchemas, ...rest }: HatchifyEverythingProps<TSchemas>): JSX.Element {
+  return (
+    <>
+      {JSON.stringify(finalSchemas) !== "{}" ? (
```

Comment:
i think the preferred way to do this is `Object.keys(finalSchemas).length !== 0` since `JSON.stringify` is slow